suricata/update/main.py

```diff
@@ -1052,7 +1052,11 @@ def _main():
     global_args, rem = global_parser.parse_known_args()
 
     if global_args.version:
-        print("suricata-update version %s (rev: %s)" % (version, revision))
+        if revision != None:
+            revision_string = " (rev: %s)" % (revision)
+        else:
+            revision_string = ""
+        print("suricata-update version %s%s" % (version, revision_string))
```

Comment:
We should also probably start using `string.format` as `%` is the older way of string formatting. See: https://docs.python.org/3.7/library/string.html#format-examples

Works since 2.7 (backported).

So we should instead have:
`print("suricata-update version {}{}".format(version, revision_string))`